scripts/magpietts/README_magpie_po.md

```diff
@@ -136,42 +136,65 @@ python scripts/magpietts/dpo/create_text_contextpairs.py \
 
```

Comment:
In addition to a readme, can we add a simple test to make sure that the online policy path works on our released checkpoints? This would mean that we have to support a path to allow for older checkpoints to work on our current branch. To address issues from other teams.

Reply:
I missed that earlier. Thanks, I have added it now. It must be mentioned somewhere, but is there a way to run CI/CD or the tests locally before pushing?

Reply:
Can you also add a reference to your new test on line 1828 on .github/workflows/cicd-main.yml?
> but is there a way to run CI/CD or the tests locally before pushing?

Yes. However I haven't found an easy to do so yet, so not yet.